arc_data_services/charts/arcdataservices/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 1.18.0
+appVersion: 1.16.0
```

Comment:
Is this intentional changing appVersion from 1.18.0 to 1.16.0?

Reply:
this should be 1.16.0. Looks like two releases ago this got bumped on accident because of a find and replace